cfgov/jinja2/v1/landing-page-2/index.html

```diff
@@ -34,6 +38,46 @@
                             'link_text' : 'Visit Google'
                             }) }}
         </div>
+
+        <div class="content-l_col-1-2">
+            {{ image_text_50_50.render({
+                            'title' : 'Image and Text 50-50',
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/434b591264048cb001bb2c998a6fbcd3f898f3d5